src/web/parserUtils.ts

```diff
@@ -49,12 +52,17 @@ function addStyling(targetElement: HTMLElement, type: MarkdownType, markdownStyl
       });
       break;
     case 'code':
-      Object.assign(node.style, markdownStyle.code);
+      Object.assign(node.style, {...markdownStyle.code, lineHeight: 1.4});
```

Comment:
I believe this is something we should discuss. We can't set the display to `inline-block` or `block` because this will break how we are positioning inline-code blocks. 

Since we're not able to do so margin won't work on inline-code block. That's why I decided to set the line-height to 1.4 - In my opinion it looks alright but let me know what do you think 😄